lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1198,6 +1195,18 @@ llvm::DIType *IRGenDebugInfo::getOrCreateDesugaredType(Type Ty,
   return getOrCreateType(BlandDbgTy);
 }
 
+unsigned IRGenDebugInfo::getObjCRuntimeVersion() {
+  // Select an ObjC runtime version to emit in the CU DIE (as
+  // dwarf::DW_AT_APPLE_major_runtime_vers) that's appropriate for the effective
+  // language version in effect for the current compilation unit.
+  assert(!IGM.Context.LangOpts.EffectiveLanguageVersion.empty());
+  if (IGM.Context.LangOpts.EffectiveLanguageVersion[0] < 2) {
```

Comment:
This is still weird, because Swift only supports the 'nonfragile' Objective-C ABI (i.e., 2), but I guess this maintains the old behavior.